### PR TITLE
[FLOC 3217] Fix broken "tutmongo" ref links 

### PR DESCRIPTION
**Pull request by davidoliverSP2**
Fixes 3217

[Review on Reviewable](https://reviewable.io/reviews/clusterhq/flocker/2031)

### CODE REVIEW

File: docs/labs/installer.rst [hunk 1]
```diff
@@ -264,7 +264,7 @@ Further reading
 
 Now that you've installed your own Flocker cluster, you may want to learn more about Flocker:
 
-* :ref:`Using Flocker <using>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
+* :ref:`Contolling Flocker <controlling-flocker>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
```
Comment (adamtheturtle) [inline docs/labs/installer.rst]: Typo "Contolling" > "Controlling"

File: docs/labs/installer.rst [hunk 2]
```diff
@@ -264,7 +264,7 @@ Further reading
 
 Now that you've installed your own Flocker cluster, you may want to learn more about Flocker:
 
-* :ref:`Using Flocker <using>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
+* :ref:`Controlling Flocker <controlling-flocker>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
```
Comment (adamtheturtle) [inline docs/labs/installer.rst]: :question: ``` :ref:`controlling-flocker` ```will render the same as ``` :ref:`Controlling Flocker <controlling-flocker>` ``` (it uses the title of the reference), and it won't risk being weird when the section title changes and this link title does not.

**Comment by adamtheturtle** [comment]
Optionally address the comment and merge when you are happy with the build results.

**Comment by davidoliverSP2** [comment]
thanks @adamtheturtle

**Pull request merged**